### PR TITLE
@orta=>Settings structure

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] Found a structure I like
- [x] Tests

### CODE REVIEW

File: Classes/Controllers/Settings/ARLabSettingsSplitViewController.m [hunk 1]
```diff
@@ -10,7 +12,22 @@ - (void)viewDidLoad
...
     self.preferredPrimaryColumnWidthFraction = 0.4;
-    
+}
+
+- (void)showDetailViewControllerForSettingsSection:(ARLabSettingsSection)section
+{
+    ARLabSettingsNavigationController *nav = [self.storyboard instantiateViewControllerWithIdentifier:SettingsNavigationController];
+    [self showDetailViewController:nav sender:self];
+
+    switch (section) {
```
Comment (sarahscott) [inline Classes/Controllers/Settings/ARLabSettingsSplitViewController.m]: if i were to use segue.identifier, i would need to implement a huge if-elseif tree, which i hate. i miss swift's string enums.

File: ArtsyFolio Tests/View Controllers/ARLabSettingsViewControllerTests.m [hunk 2]
```diff
@@ -1,43 +1,31 @@
...
-        expect(controller).to.haveValidSnapshot();
+        subject = [storyboard instantiateInitialViewController];
+        
+        nav = [subject.viewControllers find:^BOOL(id object) {
+            return [object isKindOfClass:ARLabSettingsNavigationController.class];
+        }];
+        
+        nav.managedObjectContext = context;
+        
+        expect(subject).to.haveValidSnapshot();
```
Comment (orta) [inline ArtsyFolio Tests/View Controllers/ARLabSettingsViewControllerTests.m]: much better :+1:

File: Classes/Controllers/Settings/ARLabSettingsMasterViewController.m [hunk 3]
```diff
@@ -84,17 +80,36 @@ - (void)setPresentationModeLabelText:(NSString *)string
     [self.presentationModeLabel setAttributedText:attrString];
 }
 
+#pragma mark -
+#pragma mark buttons
+- (IBAction)syncButtonPressed:(id)sender
+{
+    [(ARLabSettingsSplitViewController *)self.splitViewController showDetailViewControllerForSettingsSection:ARLabSettingsSectionSync];
```
Comment (orta) [inline Classes/Controllers/Settings/ARLabSettingsMasterViewController.m]: maybe make a function that returns `self.splitViewController` with the cast?

**Comment by orta** [comment]
:+1: rockin'

**Comment by orta** [comment]
CHANGELOG entry on next PR please

**Pull request merged**